lib/controllers/posts.js

```diff
@@ -0,0 +1,21 @@
+const { Router } = require('express');
+const authenticate = require('../middleware/authenticate');
+const Post = require('../models/Post');
+
+module.exports = Router()
+  .get('/', authenticate, async (req, res, next) => {
+    try {
+      const posts = await Post.getAll();
+      res.json(posts);
+    } catch (error) {
+      next(error);
+    }
+  })
+  .post('/', authenticate, async (req, res, next) => {
+    try {
+      const newPost = await Post.insert(req.body);
```

Comment:
Because we have access to the req.body - and the acceptance criteria mentioned a character limit - I just wanted to point out that this could be a spot to consider adding an if block, which returns an error if the post length is over 250 chars